cmd/konnector/cmd/konnector.go

```diff
@@ -33,6 +35,8 @@ import (
 	bindversion "github.com/kube-bind/kube-bind/pkg/version"
 )
 
+const LeaderElectionTimeout = 3 * time.Second
```

Comment:
isn't this a pretty short period? What's the default?

Reply:
k8s uses 20 seconds https://github.com/kubernetes/kubernetes/blob/master/cmd/kube-controller-manager/app/controllermanager.go#L206